cloudflare/resource_cloudflare_access_application.go

```diff
@@ -134,6 +142,8 @@ func resourceCloudflareAccessApplicationCreate(d *schema.ResourceData, meta inte
 		SessionDuration:        d.Get("session_duration").(string),
 		AutoRedirectToIdentity: d.Get("auto_redirect_to_identity").(bool),
 		EnableBindingCookie:    d.Get("enable_binding_cookie").(bool),
+		CustomDenyMessage:      d.Get("custom_deny_message").(string),
```

Comment:
do we always want this appended to send zero values? or should we conditionally add it in?

Reply:
We don't want to append zero values. I'm using the `omitempty` tag on those fields https://github.com/Justin-Holmes/cloudflare-go/blob/justin/access_custom_deny_fields/access_application.go#L26-L27, so I believe they'll be omitted from the request.